server/data/cache/PrisonRegisterStore.test.ts

```diff
@@ -8,54 +8,22 @@ const redisClient = {
   set: jest.fn(),
 }
 
-const activePrisons: Array<PrisonDto> = [
+const activePrisons: Array<PrisonAddress> = [
```

Comment:
The redis Prison Register Store 👇 now stores (caches) arrays of our type `PrisonAddress` rather than `PrisonDto` meaning we can do any data transformation/mapping once at the point of retrieving from the Prison Register API and caching them (ie. retrieve them as a collection of `PrisonDto` and map them into a collection of `PrisonAddress`)

Note: `PrisonAddress` is the current type name based on the previous implementation of the json file. To keep this PR as small as possible I have not refactored the name of this type or its fields, but I plan on doing that in a PR this morning